pkg/client/attach.go

```diff
@@ -296,9 +296,6 @@ func (c *ConmonClient) readStdio(
 				c.logger.Errorf("Unable to close conn: %v", connErr)
 			}
 		}
-		if cfg.Streams.Stdout != nil || cfg.Streams.Stderr != nil {
-			return <-receiveStdoutError
-		}
```

Comment:
hm I don't see the reasoning for dropping this

Reply:
In conmonrs, receiveStdoutError will never have value.  In conmonrs, even conn.Read get output result, it will never get an EOF error, so this loop will be running forever. But in conmon, after read output 'hello', it will get an EOF error, and break this loop. It seems even we finish exec command from stdin, we still need to try to read from conn ?
```
func (c *ConmonClient) redirectResponseToOutputStreams(cfg *AttachConfig, conn io.Reader) (err error) {
	buf := make([]byte, attachPacketBufSize+1) /* Sync with conmonrs ATTACH_PACKET_BUF_SIZE */
	for {
		nr, er := conn.Read(buf)
```

Reply:
even if we're absorbing the EOF [here](https://github.com/containers/conmon-rs/blob/main/pkg/client/attach.go#L244) we still want to wait to process all of the output, even if there's no input left. For instance, if the command is

`cat` and the stdin is `/dev/random\n` we'd want to see the output of /dev/random